src/main/java/dev/joopie/jambot/repository/SpotifyToYoutubeRepositoryImpl.java

```diff
@@ -0,0 +1,10 @@
+package dev.joopie.jambot.repository;
+
+import dev.joopie.jambot.models.SpotifyToYoutube;
+import dev.joopie.jambot.repository.base.BaseRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class SpotifyToYoutubeRepositoryImpl extends BaseRepository<SpotifyToYoutube> implements SpotifyToYoutubeRepository {
```

Comment:
I **think** this can be deleted due to the Autowired stuff. But I need to test this further

Reply:
Nope. It can't